src/index.tsx

```diff
@@ -1,6 +1,4 @@
 import { createRoot } from 'react-dom/client';
-import { App } from './App';
-
-createRoot(document.getElementById('root') as HTMLDivElement).render(<App />);
-
+import { Root } from './Root';
 
+createRoot(document.getElementById('root') as HTMLDivElement).render(<Root />);
```

Comment:
you can add here whole file Root.tsx 
`createRoot(document.getElementById('root') as HTMLDivElement).render(<Router>
    <Routes>
      <Route path="/" element={<App />}>
        <Route path="/" element={<h1>HomePage</h1>} />
        <Route path="home" element={<Navigate to={'/'} />} />);
        ...)
        `